docs/guide/environment.md

```diff
@@ -0,0 +1,55 @@
+# Test Environment
+
+Vitest provides [`environment`](/config/#environment) option to run code inside a specific environment. You can modify how environment behaves with [`environmentOptions`](/config/#environmentoptions) option.
+
+By default, you can use these environments:
+
+- `node` is default environment
+- `jsdon` emulates browser environment by providing Browser API, uses [`jsdom`](https://github.com/jsdom/jsdom) package
+- `happy-dom` emulates browser environment by providing Browser API, and considered to be faster than jsdom, but lacks some API, uses [`happy-dom`](https://github.com/capricorn86/happy-dom) package
+- `edge-runtime` emulates Vercel's [edge-runtime](https://edge-runtime.vercel.app/), uses [`@edge-runtime/vm`](https://www.npmjs.com/package/@edge-runtime/vm) package
+
+Starting from 0.23.0, you can create your own package to extend Vitest environment. To do so, create package with the name `vitest-environment-${name}`. That package should export an object with the shape of `Environment`:
+
+```ts
+import type { Environment } from 'vitest'
+
+export default <Environment>{
+  name: 'custom',
+  setup() {
+    // cusotm setup
```

Comment:
```suggestion
    // custom setup
```

Reply:
It's fixed already:

https://github.com/sheremet-va/vitest/blob/27f947a4ba40eb277fce3aa01d0225d5105f46fe/docs/config/index.md#L248

What commit are you watching?

Reply:
That was in docs/config/index.md

Reply:
Oh, I didn't notice